src/quest_manager/models.py

```diff
@@ -30,6 +30,13 @@ class Category(IsAPrereqMixin, models.Model):
         help_text="Quests that are a part of an inactive campaign won't appear on quest maps and won't be available to students."
     )
 
+    import_id = models.UUIDField(
+        default=uuid.uuid4,
+        blank=True,
```

Comment:
`blank=False`, this should never be blank, it's a required field.  blank=True while null=False is just asking for problems with our database I think?